project-words.txt

```diff
@@ -518,3 +518,9 @@ grafanacon
 upsert
 Dedup
 dedup
+PROCID
+MSGID
+SSSZ
+NILVALUE
+PROCID
+MSGID
```

Comment:
Why do we add these?

Reply:
The comment I added to document the RFC5424 spec tripped the spelling czar

```
// Format the RFC5424 message - strictly adhering to RFC5424 format
// <PRI>VERSION TIMESTAMP HOSTNAME APP-NAME PROCID MSGID STRUCTURED-DATA MSG
```